### DISCUSSION TITLE
Add the option to hide class attribute in vscode plugin

### DISCUSSION THREAD

**Discussion by kushttrim** [tailwindlabs/tailwindcss#7922] [Ideas]
Hello,

Not sure if this has been suggested before but I could not find anything.

One thing I dislike about tailwind is how much it pollutes the html class attribute with its names.
Most of my time is not actually spent in css and seeing these tags just makes it look really bad.

![1](https://user-images.githubusercontent.com/31842558/159665863-bb296fb3-2039-46a5-b017-0712f80bf15d.jpg)

How about having an option in vscode plugin to automatically hide class names when tailwind is detected and when you click it you can toggle visibility? This would make html much readable and easier to digest.

![2](https://user-images.githubusercontent.com/31842558/159665877-1e7efb99-8ad6-4511-a6fb-d38502be5f76.jpg)

**Comment by Erel0** [comment]
great idea

**Comment by LucasKuhn** [comment]
There is an extension called [Inline Fold](https://github.com/moalamri/vscode-inline-fold) that swaps the class attribute for dots `...` 
Someone could use that as en example to build something into the tailwind plugin

**Reply by matheuscastroks** [reply]
Nossa senhora, mto obrigado!

**Comment by LauGau** [comment]
I would love to see that too ! :)

**Reply by devpishaili** [reply]
Hello. Does this currently have support for PHP Developers?

**Comment by stivoat** [comment]
Here you go! 
[Tailwind Fold](https://marketplace.visualstudio.com/items?itemName=stivo.tailwind-fold)

I drew inspiration from your design :)

**Reply by Kris-Pelteshki** [reply]
Love it! ❤️

**Reply by donmb1** [reply]
Really nice. As an improvement I'd suggest to use some kind of Retina logo. It looks a bit blurry on Retina displays
<img width="299" alt="Screenshot 2023-01-24 at 13 09 04" src="https://user-images.githubusercontent.com/3296757/214288029-5541bbe1-41e1-424f-9118-af500bb5c0e6.png">

**Comment by Helpico** [comment]
There is a custom decorator "Inline fold" in VSC.

**Comment by pablojsx** [comment]
thanks! i'm using it to just set the opacity to .5 haha
